graphene_django/utils/testing.py

```diff
@@ -128,7 +128,7 @@ def _client(self, client):
         )
         self.client = client
 
-    def assertResponseNoErrors(self, resp, msg=None):
+    def assert_response_no_errors(self, resp, msg=None):
```

Comment:
This is a breaking change for consumers of this package. I would recommend at least preserving an alias with the original camelCase name.  I imagine it was named this way for consistency with Django (and Python's) camelCase assertion methods: https://docs.djangoproject.com/en/4.2/topics/testing/tools/#assertions